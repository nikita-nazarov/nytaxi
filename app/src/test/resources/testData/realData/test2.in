1588222685000000,1588224164000000