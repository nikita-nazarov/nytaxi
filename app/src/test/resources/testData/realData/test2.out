0,2.4
1,1.29
2,4.73
3,4.93